includes/admin/settings/class-settings-gateways.php

```diff
@@ -344,58 +344,62 @@ public function render_enabled_gateways( $field, $settings ) {
-					esc_attr( $key ),
-					esc_html( $label ),
-					esc_html( $option['checkout_label'] )
-				);
-
-				printf(
-					'<input class="gateways-radio" type="radio" name="%1$s" value="%2$s" %3$s %4$s>',
-					'default_gateway',
-					$key,
-					checked( $key, $default_gateway, false ),
-					disabled( null, $enabled, false )
-				);
-
-				printf(
-					'<input class="gateways-checkbox" name="%1$s[%2$s]" id="%1$s[%2$s]" type="checkbox" value="1" %3$s data-payment-gateway="%4$s"/>',
-					esc_attr( $id ),
-					esc_attr( $key ),
-					checked( '1', $enabled, false ),
-					esc_html( $option['admin_label'] )
-				);
-				echo '</li>';
-			endforeach;
-			echo '</ul>';
-
-			echo '</div>'; // end gateway-enabled-wrap.
-
-			printf(
-				'<tr><th>%1$s</th><td>%2$s</td></tr>',
-				$field['title'],
-				ob_get_clean()
-			);
-		}
+                if (is_array($settings) && array_key_exists($key, $settings)) {
+                    $enabled = '1';
+                }
+
+                echo '<li>';
+                printf('<span class="give-drag-handle"><span class="dashicons dashicons-menu"></span></span>');
+                printf(
+                    '<span class="admin-label">%1$s %2$s</span>',
+                    esc_html($option['admin_label']),
+                    !empty($option['admin_tooltip']) ? Give()->tooltips->render_help(
+                        esc_attr($option['admin_tooltip'])
+                    ) : ''
+                );
+
+                $label = '';
+                if (!empty($gateways_label[$key])) {
+                    $label = $gateways_label[$key];
+                }
+
+                printf(
+                    '<input class="checkout-label" type="text" id="%1$s[%2$s]" name="%1$s[%2$s]" value="%3$s" placeholder="%4$s"/>',
+                    'gateways_label',
+                    esc_attr($key),
+                    esc_html($label),
+                    esc_html($option['checkout_label'])
+                );
+
+                printf(
+                    '<input class="gateways-radio" type="radio" name="%1$s" value="%2$s" %3$s %4$s>',
+                    'default_gateway',
+                    $key,
+                    checked($key, $default_gateway, false),
+                    disabled(null, $enabled, false)
+                );
+
+                printf(
+                    '<input class="gateways-checkbox" name="%1$s[%2$s]" id="%1$s[%2$s]" type="checkbox" value="1" %3$s data-payment-gateway="%4$s"/>',
+                    esc_attr($id),
+                    esc_attr($key),
+                    checked('1', $enabled, false),
+                    esc_html($option['admin_label'])
+                );
+                echo '</li>';
+            endforeach;
+            echo '</ul>';
+
+            echo '</div>'; // end gateway-enabled-wrap.
+
+            echo '<div style="padding-top: 1rem;"><p><sup>*</sup>(v2) This gateway is only supported on GiveWP version 2.x.x donation forms (aka v2 forms)</p></div>';
```

Comment:
Let's coordinate with Support/docs to have a link folks can click on to learn more about what this means on givewp.com.

Reply:
I don't think a doc exists at this point, so it may mean adding a task to the backlog and starting coordination with @Benunc.

Reply:
will do